### DISCUSSION TITLE
Scrolling settings is bad experience.

### DISCUSSION THREAD

**Discussion by KoleckOLP** [ppy/osu#15842] [Ideas] [answered]
I think the design of the new setting menu should change.

When you are casually scrolling, it happens very often that the part of the settings that covers 75% of the screen is grayed out and the 25% left are highlighted and usable.

examples:
![osu!_gQFXhiVb1l](https://user-images.githubusercontent.com/22729896/143696700-2cd66708-d209-476e-94ec-a897150a1323.png)

![image](https://user-images.githubusercontent.com/22729896/143703646-2ac73963-0530-4a17-bd05-45dbf407f807.png)

It makes browsing the setting menu really hard for me. I can never find what I am looking for by scrolling, I do not remember what category are things in and can't tell the categories apart by their icons. This does not have to change, but please make the part that is on screen highlighted.

It seems that the game prefers to highlight the top category of the settings even though bottom category takes up more space, and I personally look in the middle of the settings not the top.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/14414

**Comment by peppy** [comment]
As mentioned in the linked issue, the centre point is wrong (and the code surrounding it is incomprehensible and needs a rewrite).
